content/common/contributing-docs.md

```diff
@@ -4,23 +4,28 @@ date = "2022-01-01T00:00:01Z"
 [extra]
 
 ---
-
 - [Technical Documentation Types](#technical-documentation-types)
   - [1. Tutorials](#1-tutorials)
   - [2. How-To Guides](#2-how-to-guides)
   - [3. Reference](#3-reference)
   - [4. Explanation](#4-explanation)
+- [Documents Relevant to Two or More Projects](#documents-relevant-to-two-or-more-projects)
 - [Technical Documentation Procedure](#technical-documentation-procedure)
   - [1. Fork the Repository](#1-fork-the-repository)
   - [2. Clone the Fork](#2-clone-the-fork)
   - [3. Create New Branch](#3-create-new-branch)
   - [4. Add Upstream](#4-add-upstream)
   - [5. Code Blocks, Annotations and Table of Contents (ToC)](#5-code-blocks-annotations-and-table-of-contents-toc)
-  - [6. Check Content](#6-check-content)
-  - [7. Add Changes](#7-add-changes)
-  - [8. Commit Changes](#8-commit-changes)
-  - [9. Push Changes](#9-push-changes)
-  - [10. Create a Pull Request](#10-create-a-pull-request)
+  - [Implementing a Table of Contents (ToC)](#implementing-a-table-of-contents-toc)
```

Comment:
should we add `6` here as count?

Reply:
Ah, thanks, fixing now (that is supposed to be bold only (because I already mention ToC in the previous heading on top). Nice catch!